app/engine/tusk.py

```diff
@@ -191,9 +191,9 @@ def create_enemies(self) -> None:
 
         # Change max hp based on player count
         max_hp = {
-            1: 300,
-            2: 500,
-            3: 800
+            1: 200,
+            2: 300,
+            3: 400
```

Comment:
Why did you change tusk's health?

Reply:
My brother thought it was too hard, and it was taking a very long time to get his health down;
you can change it back if you like